streamalert/classifier/clients/firehose.py

```diff
@@ -298,6 +312,46 @@ def firehose_log_name(cls, log_name):
         """
         return re.sub(cls.SPECIAL_CHAR_REGEX, cls.SPECIAL_CHAR_SUB, log_name)
 
+    @classmethod
+    def generate_firehose_stream_name(cls, use_prefix, prefix, log_stream_name):
```

Comment:
This method's name seems to imply that it generates the full firehose name, but that's actually not true right? It generates the **prefix-less** firehose name.

I suggest renaming the function to `generate_firehose_suffix` reduce confusion

Reply:
Good point! Yes, you are right, the return result doesn't include prefix. Will rename this method.